x-pack/plugins/reporting/server/routes/lib/request_handler.ts

```diff
@@ -71,7 +71,7 @@ export class RequestHandler {
     const payload = {
       ...job,
       headers,
-      title: jobParams.title,
+      title: job.title,
```

Comment:
csv v2 failures in title were because of pulling the title from jobParams vs the job that results from the createJob(). Looked locally, and doesn't impact other export types